Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM ubuntu:xenial
+# Ubuntu Xenial(16.04) with Swift 5.3.1 installed
+FROM swift:5.3.1-xenial
```

Comment:
Is there another way to do this besides changing the base image? We probably do not want to change the entire image just to support a single package manager. There are also cases where other images are built off of LF and these cases would need to be tested by anyone that is doing that if were to make this change. It can be done but thinking there has to be a different way the required things in the swift image can be installed

Reply:
I am not an expert on Docker, so any suggestions welcome. It was the easiest way for me to try add support for SPM, because it was also based on Ubuntu 16.04. If someone can share what would be the best way to do it, I can give it a try soon.

Reply:
I can give it a try with https://docs.docker.com/develop/develop-images/multistage-build/

Reply:
This is less of a docker question and more of a question of how install SPM. Looking [here](https://github.com/apple/swift-package-manager#installation), there are steps to install it and I am wondering if they will work. If you look in the Dockerfile, that is how other package managers are being installed. For testing purposes, what you can do is boot into the image then run the installation commands and check if it does install the swift cli properly. Once you know, you can add the steps to the docker image and then rebuild it locally to try it out

Reply:
We could do that, it would basically be a copy paste from here: https://github.com/apple/swift-docker/blob/main/5.3/ubuntu/16.04/Dockerfile
Maybe to keep shorter gpg signature verification could be omitted. But those instructions for Linux would basically be it.
If copy-paste from there is ok, I can do it. 
Or could use a multistage build - and then do we want it is probably a Docker question. I am not aware of less obvious pros and cons, except that for future Swift versions it would be much easier to maintain multistage build, basically delegating the process of Swift installation to a Dockerfile maintained by Apple.

Reply:
Copying and pasting is not a bad starting point for this. If you take a look at the LF dockerfile, we already have most package managers hard coded to a version at the top. The only issue I see happening is if the signing key changes and I don't know if there is an automated way to get it. If you look at the composer section, there is an automated way there to get keys